docs/getting-started/server/self-hosted/index.md

```diff
@@ -137,23 +137,23 @@ development.
 
 If you followed [Server Setup Guide](../guide.md#create-database) when creating your
 cloud-configured database, you just need to run the same PowerShell migration script with an
-argument of `-s`:
+argument of `-selfhost`:
```

Comment:
the script was complaining about `-s` being too ambiguous; used `-selfhost` successfully